packages/forms/src/styled/radio/StyledRadioInput.spec.js

```diff
@@ -6,13 +6,40 @@
  */
 
 import React from 'react';
-import { render } from 'garden-test-utils';
-import StyledRadioInput from './StyledRadioInput';
+import { render, renderRtl } from 'garden-test-utils';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import { StyledRadioInput } from './StyledRadioInput';
+import { StyledRadioLabel } from './StyledRadioLabel';
 
 describe('StyledRadioInput', () => {
-  it('renders default styling correctly', () => {
+  it('renders the expected element', () => {
     const { container } = render(<StyledRadioInput />);
 
+    expect(container.firstChild.nodeName).toBe('INPUT');
     expect(container.firstChild).toHaveAttribute('type', 'radio');
   });
+
+  it('renders expected checked styling', () => {
+    const { container } = render(<StyledRadioInput defaultChecked />);
+
+    expect(container.firstChild).toHaveStyleRule('background-color', PALETTE.blue[600], {
```

Comment:
Open question, should we be asserting that our elements have applied the _correct color_ or the _correct hue_?

If we are saying that the element should be applying `primary600` it may make more sense to assert:

```js
expect(container.firstChild).toHaveSytleRule('background-color', getColor('primaryHue', 600, DEFAULT_THEME));
```

Reply:
I think you're right. For example if `primaryHue` changes some day, we don't want all these tests to break. Just me ... being lazy. I'll get these spruced up.